samples/benchmarks/coremark/sample.yaml

```diff
@@ -61,8 +86,8 @@ tests:
       - nrf54l15dk/nrf54l15/cpuapp
       - nrf54h20dk/nrf54h20/cpuapp
     tags: ci_build sysbuild ci_samples_benchmarks
-    extra_args: EXTRA_CONF_FILE="prj_static_memory.conf"
-  sample.benchmark.coremark_multithread:
+    extra_args: FILE_SUFFIX=static_memory
+  sample.benchmark.coremark.multiple_threads:
```

Comment:
Did you check correct working of those - I guess that there might be some problems with heap configurations on ppr core.

Reply:
yes, they work. I don't understand why the heap configuration wouldn't work for PPR core.

Also, we are discussing heap configuration in the comment that was attached to the line with "multiple_threads" configuration